spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -529,6 +529,46 @@ Spring Cloud Data Flow Dashboard or RESTful API), will relaunch  the
 `STOPPED` child task, and then launch the remaining (unlaunched) child tasks
 in the specified order.
 
+==== Launching a Composed Task From a Stream
```

Comment:
What if we move this under "Launching Tasks from a Stream" section? That way, all the stream launching task examples could be all together. _(I had a reordering suggestion in the acceptance)_